controller.go

```diff
@@ -798,9 +798,11 @@ func (c *controller) NewNetwork(networkType, name string, id string, options ...
 		if err != nil {
 			return nil, types.NotFoundErrorf("configuration network %q does not exist", network.configFrom)
 		}
+		internal := network.internal
 		if err = t.applyConfigurationTo(network); err != nil {
 			return nil, types.InternalErrorf("Failed to apply configuration: %v", err)
 		}
+		network.generic[netlabel.Internal] = internal
```

Comment:
why do we create new variable internal ?  Instead can't we directly use network.internal?

Reply:
Sorry, I think we can.
I just got little bit lost in unknown sources.